x-pack/plugins/security_solution/public/management/components/policy_response/policy_response_friendly_names.ts

```diff
@@ -405,9 +420,24 @@ const linkTexts = Object.freeze(
         }
       ),
     ],
+    [
+      'linux_deadlock',
+      i18n.translate(
+        'xpack.securitySolution.endpoint.details.policyResponse.link.text.linux_deadlock',
+        {
+          defaultMessage: ' Learn more.',
+        }
+      ),
+    ],
   ])
 );
 
+export const LINUX_DEADLOCK_MESSAGE = 'Disabled due to potential system deadlock';
```

Comment:
Maybe move this above line 370 where you declare `descriptions` and use this const as `defaultMessage` for `xpack.securitySolution.endpoint.details.policyResponse.description.linux_deadlock` key

Reply:
I agree that this is a bit wasteful. The reason I had to do this is because the description one actually has a period at the end. Unfortunately, can't use string interpolation inside `i18n.translate` 😞 . Though we're most likely going to change the description text anyways.